.github/workflows/maven.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Checkout network store sources
+        uses: actions/checkout@v1
+        with:
+          repository: powsybl/powsybl-network-store
+          ref: refs/heads/master
+
+      - name: Build and install network store client with Maven
+        run: mvn --batch-mode -Pclient -DskipTests=true --file ../powsybl-network-store/pom.xml install
+
+      - name: Checkout sources
+        uses: actions/checkout@v1
+
+      - name: Build with Maven
+        run: mvn --batch-mode package
+
+      - name: Run SonarCloud analysis
+        run: >
+          mvn --batch-mode -Pjacoco package sonar:sonar
+          -Dsonar.host.url=https://sonarcloud.io
+          -Dsonar.organization=powsybl-ci-github
+          -Dsonar.projectKey=com.powsybl:powsybl-network-map-server
```

Comment:
shouldn't we add that part on another PR so it doesn't mark the build as failed ?

Reply:
I found a solution to this problem. We just have to pre-create project on Sonar Cloud side before running the CI and there is no more error.